src/main/java/org/sagebionetworks/workers/util/aws/message/PollingMessageReceiverImpl.java

```diff
@@ -110,6 +110,20 @@ public PollingMessageReceiverImpl(AmazonSQSClient amazonSQSClient,
 	 */
 	@Override
 	public void run(final ProgressCallback<Message> containerProgressCallback) throws Exception {
+		Message message = null;
+		do {
+			message = pollForMessage();

```

Comment:
That was the intention.

The trigger firing frequency was not intended to be a throttle on the speed that messages could be processed.  Instead, the trigger frequency should be a balance between fast message response time (high frequency) and low resource utilization when there is no work to do (low frequency).  As long as there is work to be done then we want to do it ASAP.